infercnvpy/tl/_copykat.py

```diff
@@ -0,0 +1,106 @@
+import infercnvpy as cnv
+import numpy as np
+import pandas as pd
+import scipy
+from scipy.sparse import issparse
+from anndata import AnnData
```

Comment:
duplicate import of anndata. Doesnt matter which, they point to the same object in the end.

Reply:
my bad